README.md

```diff
@@ -12,10 +12,12 @@ template for creating websites for workshops.
     [automatically published as a website by GitHub][github-project-pages].
 
 3.  Once you are done,
-    please **send your repository's URL to the [Software Carpentry administrator][contact]**.
+    please access [AMY's self-organized workshop form][self-organized-workshop-form],
+    fill it and Software Carpentry administrator will contact you back
+    if need any extra information.
     We build the list of workshops on our websites from the data included in your `index.html` page.
     We can only do that if you [customize][customization] that page correctly
-    *and* send us a link to your workshop website.
+    *and* [fill the form][self-organized-workshop-form].
```

Comment:
"[fill in the form][...]" - sorry, there's no good reason for English to say "fill in" instead of "fill", but we do.